### PR TITLE
Neatened project for windows release.

### PR DESCRIPTION
**Pull request by KlaussMC**
There, I had to close the previous pull request because I wasn't able to create this one, I have completed what you had asked. I also did another quick test. nothing seems to have broken, which is good...

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -9,6 +9,8 @@ import './index.css';
 import fileIcon from './img/file.png';
 import arrow from './img/arrow.png';
 import folderIcon from './img/folder.png';
+// const fs = require("fs");
```
Comment (eladnava) [inline src/index.js]: Please remove these unused commented imports 👍

File: src/util/adb.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ const fs = electron.remote.require('fs');
 const path = electron.remote.require('path');
 const adb = electron.remote.require('adbkit');
 const moment = electron.remote.require('moment');
-const spawn = electron.remote.require('child_process').spawn;
+const { spawn } = electron.remote.require('child_process');
```
Comment (eladnava) [inline src/util/adb.js]: Was there actually a difference when you changed the syntax this way? Does the former require not work?

File: src/util/adb.js [hunk 3]
```diff
@@ -200,5 +200,6 @@ export default {
 
 function pathToAdb() {
     // Resolve path to adb binary (bundled with pixelmate)
-    return path.join(`${app.getAppPath()}/bin/adb`);
-}
\ No newline at end of file
+	if (window.require('os').platform() === "darwin") return path.join(`${app.getAppPath()}/bin/adb`);
```
Comment (eladnava) [inline src/util/adb.js]: Can you please format these lines according to the if-else convention in the rest of the project? Thx 👍

File: package.json [hunk 4]
```diff
@@ -14,12 +14,14 @@
...
+      "start":"electron .",
+    "darwin": "URL=http://localhost:3000 electron",
+    "win32":"set URL=http://localhost:3000 && electron .",
     "server": "react-scripts start",
     "build": "react-scripts build",
     "package": "rm -rf dist/* build/* && npm run build && electron-packager . Pixelmate --electron-version=1.7.11 --platform=darwin --arch=x64 --icon=public/app.icns --overwrite && zip -r -y -X dist/Pixelmate.zip Pixelmate-darwin-x64/Pixelmate.app"
   },
   "devDependencies": {
-    "react-scripts": "1.1.0"
+    "react-scripts": "^1.1.4"
```
Comment (eladnava) [inline package.json]: Any specific reason to update `react-scripts`? Does it not work with version `1.1.0`?

**Comment by eladnava** [comment]
By the way, if you edit your changes in the branch of this PR, and sync them with GitHub, it will update the PR automatically.

File: src/index.js (hunk 1 above)
Comment (KlaussMC) [reply src/index.js]: you know, I feel like an Idiot now. 😂

Comment (eladnava) [reply src/index.js]: Nah, happens to everyone 😝

File: src/util/adb.js (hunk 2 above)
Comment (KlaussMC) [reply src/util/adb.js]: IDK, it seems to work best this way, I don't know why, but then again, we've had some strange behaviour lately, right?

File: src/util/adb.js (hunk 3 above)
Comment (KlaussMC) [reply src/util/adb.js]: I did try to follow your conventions as much as possible, I simply forgot.

File: src/util/adb.js (hunk 2 above)
Comment (eladnava) [reply src/util/adb.js]: Any chance you can test it with the former to see if it still works? Might have been another factor causing it not to work when you changed it.

File: package.json (hunk 4 above)
Comment (KlaussMC) [reply package.json]: I didn't notice that. no reason at all. which do you think is best?

File: src/util/adb.js (hunk 3 above)
Comment (eladnava) [reply src/util/adb.js]: No harm done.

File: package.json (hunk 4 above)
Comment (eladnava) [reply package.json]: The original 👍  According to the philosophy, 'don't fix it if it ain't broke' 🤣

Comment (KlaussMC) [reply package.json]: I left that, but yes, I'll revert that.

**Comment by KlaussMC** [comment]
> Sync them with GitHub

you just mean push the changes?

**Comment by eladnava** [comment]
@KlaussMC What about all the `img/` file commits? None of the images are being referenced currently. If this is for a future commit please exclude these from the current PR.

**Comment by KlaussMC** [comment]
These files will not be included when you run `npm run build` will they? you could run that and have the files not included automatically?

File: package.json [hunk 5]
```diff
@@ -14,12 +14,14 @@
     "react-dom": "^16.2.0"
   },
   "scripts": {
-    "app": "URL=http://localhost:3000 electron .",
+      "start":"electron .",
```
Comment (eladnava) [inline package.json]: Please remove excessive leading whitespace.

File: package.json [hunk 6]
```diff
@@ -14,12 +14,14 @@
...
+    "darwin": "URL=http://localhost:3000 electron",
+    "win32":"set URL=http://localhost:3000 && electron .",
     "server": "react-scripts start",
     "build": "react-scripts build",
     "package": "rm -rf dist/* build/* && npm run build && electron-packager . Pixelmate --electron-version=1.7.11 --platform=darwin --arch=x64 --icon=public/app.icns --overwrite && zip -r -y -X dist/Pixelmate.zip Pixelmate-darwin-x64/Pixelmate.app"
   },
   "devDependencies": {
-    "react-scripts": "1.1.0"
+    "react-scripts": "^1.1.0"
```
Comment (eladnava) [inline package.json]: Please remove `^` as it would be better to leave things as they are if they are working. 👍

Comment (KlaussMC) [reply package.json]: yes, if it aint broke, don't fix it

**Comment by KlaussMC** [comment]
All done (I hope)

File: package.json [hunk 7]
```diff
@@ -7,19 +7,21 @@
   "homepage": "./",
   "main": "src/electron.js",
   "dependencies": {
-    "adbkit": "^2.11.0",
```
Comment (eladnava) [inline package.json]: Why did you remove the `^` from all of these dependencies?

If you look at the "Files changed" tab in the PR the changes listed should only be related to solving the PR's goal. 👍

File: package.json [hunk 8]
```diff
@@ -7,19 +7,21 @@
...
+    "start":"electron .",
+    "darwin": "URL=http://localhost:3000 electron",
+    "win32":"set URL=http://localhost:3000 && electron .",
     "server": "react-scripts start",
     "build": "react-scripts build",
     "package": "rm -rf dist/* build/* && npm run build && electron-packager . Pixelmate --electron-version=1.7.11 --platform=darwin --arch=x64 --icon=public/app.icns --overwrite && zip -r -y -X dist/Pixelmate.zip Pixelmate-darwin-x64/Pixelmate.app"
   },
   "devDependencies": {
-    "react-scripts": "1.1.0"
+    "react-scripts": "^1.1.0"
```
Comment (eladnava) [inline package.json]: It's still there 🤣

Comment (KlaussMC) [reply package.json]: Oh, sorry, that was a silly mistake. I should have known. I'll go fix that!

**Comment by KlaussMC** [comment]
Well, I'm happy we got that working, we'll now have to update the README so that people know it's on windows, should I do that?

**Comment by eladnava** [comment]
Yes, absolutely 👍

**Comment by eladnava** [comment]
Yes, exactly.

**Comment by KlaussMC** [comment]
Okay. Sorry for constantly asking questions, I only recently understood the benefit of Git, now I'm beginning to see how awesome GitHub really is!

**Comment by KlaussMC** [comment]
There, new PR is open.
